### DISCUSSION TITLE
Possible to provide source changes?

### DISCUSSION THREAD

**Discussion by FroggieFrog** [synman/Ender-3-S1-Pro-Firmware#47] [General]
Is it possible to provide the actual sources your firmware is compiled with?
Or even just your changes/adjustments?

It would be more helpful than just providing the compiled binaries to find bugs or even help fixing them.

**Comment by pacocatech** [comment]
I would also really like to have access to the source code, especially to see how the correction related to the M600 command was, that the printer does not wait for the user to choose to return and it returns by itself.

can you share the source with us?

**Reply by bvweerd** [reply]
Just one of synman's other repos: https://github.com/synman/Marlin/tree/2.1.2-ender-3-s1

**Reply by pacocatech** [reply]
thanks a lot man! 

Do I need to update the display?

**Reply by bvweerd** [reply]
Nope, at least I didn't. Works as is so far I can see

**Reply by pacocatech** [reply]
amazing!
My biggest problem today is the printer not respecting the M600 command, it makes the return by itself. I will test this version and understand what was fixed.
Once again thank you very much for sharing.
